lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/DBORevision.java

```diff
@@ -199,9 +210,10 @@ public MigrationType getMigratableTableType() {
 			public DBORevision createDatabaseObjectFromBackup(DBORevision backup) {
 				try {
 					// Retrieve the historical reference from a blob
-					Reference ref = DBORevisionUtils.convertBlobToReference(backup.getReference());
+					Reference ref = DBORevisionUtils.convertBlobToReference(backup.getReferences());
 					byte[] blob = JDOSecondaryPropertyUtils.compressReference(ref);

```

Comment:
While this works it means we must change this in release-100 or we will lose all references when we migrate from 99 to 100.  It would be safer to use references only if reference was null.